src/Meshes.jl

```diff
@@ -296,6 +296,8 @@ export
   FaceTouchingBoxes,
   CornerTouchingBoxes,
   IntersectingSegmentTriangle,
+  IntersectingSegmentPlane,
+  ContainedSegmentPlane,
```

Comment:
Maybe we should try to follow the naming convention of other intersections? Would it make sense to call these `CrossingSegmentPlane` and `OverlappingSegmentPlane` instead?